src/rf/js/src/home/views.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var React = require('react'),
+    asset = require('../core/utils').asset;
+
+var LoginView = React.createBackboneClass({
+    render: function() {
+        return <div>
+            <div className="block login-block">
+                <img src={asset('img/logo-raster-foundry.png')} />
+                <div className="form-group">
+                    <label htmlFor="search">Username</label>
+                    <input type="text" className="form-control" id="search" />
+                </div>
+                <div className="form-group">
+                    <label htmlFor="login-password">Password</label>
+                    <input type="password" className="form-control" id="login-password" />
+                </div>
+                <div className="form-action">
+                    <a href="#" className="btn btn-secondary btn-block"
+                        onClick={this.props.handleLogin}>Log In</a>
```

Comment:
I think if we use actual form tags to surround these form elements then we can use onSubmit which should capture the enter key as well as clicking the button. Also, I think we've been doing  a bad job recently of using buttons and links. They're not supposed to be interchangeable. We should use buttons for forms and links for things that actually only change the URL but don't submit data anywhere.